src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -1013,9 +1013,9 @@ private void ResetWindowState()
             suppress_resize--;
         }
 
-        private void GrabCursor(bool grab)
+        private void GrabCursor(bool shouldGrab)
```

Comment:
I've got similar concerns as HideCursor for this method. It should be renamed or split.